libqtile/widget/base.py

```diff
@@ -386,7 +386,11 @@ class _TextBox(_Widget):
         ("foreground", "ffffff", "Foreground colour"),
         ("fontshadow", None, "font shadow color, default is None(no shadow)"),
         ("markup", True, "Whether or not to use pango markup"),
-        ("fmt", "{}", "How to format the text"),
+        (
+            "fmt",
+            "{}",
+            "To format the string returned by the widget. For example, if the clock widget returns '08:46' we can do fmt='time {}' do print 'time 08:46' on the widget. To format the individual strings like hour and minutes use the format paramater of the widget (if it has one)",
```

Comment:
You need to run black to format the code. It should sort this out for you.

Reply:
Black doesn't seem to want to change this. I've manually split it up into four lines.

Reply:
Huh, interesting. Cool, thanks.